### PR TITLE
feat: add image and input components

### PR DESCRIPTION
**Pull request by RakshaPawar108**
Added Image and Input components. Please review and give feedback and suggestions.

Adding screenshots for easy viewing

Screenshot (104)

Screenshot (105)

Screenshot (106)

Screenshot (107)

Screenshot (108)

### CODE REVIEW

**Review by mohammed-sas — APPROVED** [review]
good to merge

**Comment by AnkurChunekar** [comment]
Great work Raksha, I have one feedback, your responsive image is not adjusting according to the size of the container, i think you have explicitly given it a fixed height can you look into this.

File: components/Image/image.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+.img-responsive {
+	height: 100%;
+	width: 100%;
+}
+.img-container,
+.img-round,
+.img-rounded-corner,
+.img-responsive {
+	object-fit: cover;
+}
```
Comment (Kevin-Solomon) [inline components/Image/image.css]: This code is awesome.
But a small suggestion would be to have a common *img* class having common properties.

**Review by Kevin-Solomon — APPROVED** [review]

File: components/Image/image.css (hunk 1 above)
Comment (RakshaPawar108) [reply components/Image/image.css]: Done

**Pull request merged**